src/dra/75F54.c

```diff
@@ -2911,7 +2911,8 @@ void func_8012CB4C(void) {
 void func_8012CC30(s32 arg0) {
     if (arg0 == 0) {
         D_80138444 = 1;
-        if (D_80138FC0[1].x == 255 && IsRelicActive(RELIC_SKILL_OF_WOLF) &&
+        if (g_buttonCombo[1].buttons_correct == (u8)(-1) &&
```

Comment:
It seems like each entry represents a spell, so we could do `g_buttonCombo[COMBO_WOLF_CHARGE].buttons_correct`

Reply:
Just to be clear, you'd like me to create a new enum to hold all the button combos?

Reply:
Yeah I was thinking that would make sense to help document